request-options.js

```diff
@@ -1,13 +1,14 @@
 let config = require('./config/config');
+const fs = require('fs');
 
 function getRequestOptions(options) {
     return {
         strictSSL: config.request.rejectUnauthorized,
         json: true,
         headers: { 'ph-auth-token': options.token },
-        ca: config.request.ca,
+        ca: fs.readFileSync(config.request.ca),
         proxy: config.request.proxy,
-        cert: config.request.cert,
+        cert: fs.readFileSync(config.request.cert),
```

Comment:
Are these parameters being validated anywhere to ensure that it exists in the config file?  For example, in most of our integrations we check to make sure `config.request.cert` (along with all the other settings), are present and have a valid value:
```
  if (typeof config.request.cert === 'string' && config.request.cert.length > 0) {
    defaults.cert = fs.readFileSync(config.request.cert);
  }
```